components/desktop/mate/mate-common/Makefile

```diff
@@ -15,33 +15,33 @@
 # Copyright 2019 Michal Nowak
 #
 
-PREFERRED_BITS=64
+BUILD_BITS=		64
```

Comment:
Do I understand correctly, that there's no difference is it build 32-bit or 64-bit, as these are just some scripts?

Reply:
Yes, it seems:

```diff
 newman  ~  gdiff -pruN old/ new/
diff -pruN old/mate-autogen.1 new/mate-autogen.1
--- old/mate-autogen.1	2019-10-04 06:41:03.884689759 +0000
+++ new/mate-autogen.1	2019-10-04 06:29:08.437234681 +0000
@@ -44,6 +44,6 @@ Perberos <perberos@gmail.com>
 .SH NOTES
 
 .\" Oracle has added source availability information to this manual page
-This software was built from source available at https://openindiana.org/.  The original community source was downloaded from  http://pub.mate-desktop.org/releases/1.22/mate-common-1.22.0.tar.xz
+This software was built from source available at https://openindiana.org/.  The original community source was downloaded from  http://pub.mate-desktop.org/releases/1.22/mate-common-1.22.2.tar.xz
 
 Further information about this software can be found on the open source community website at http://www.mate-desktop.org.
diff -pruN old/mate-doc-common new/mate-doc-common
--- old/mate-doc-common	2019-10-04 06:41:03.872948752 +0000
+++ new/mate-doc-common	2019-10-04 06:29:08.426094455 +0000
@@ -4,7 +4,7 @@ progname=`echo "$0" | sed 's%^.*/%%'`
 
 PROGRAM=mate-doc-common
 PACKAGE=mate-common
-VERSION=1.22.0
+VERSION=1.22.2
 
 prefix=/usr
 datarootdir=${prefix}/share
diff -pruN old/mate-doc-common.1 new/mate-doc-common.1
--- old/mate-doc-common.1	2019-10-04 06:41:03.888905666 +0000
+++ new/mate-doc-common.1	2019-10-04 06:29:08.439678985 +0000
@@ -63,6 +63,6 @@ Perberos <perberos@gmail.com>
 .SH NOTES
 
 .\" Oracle has added source availability information to this manual page
-This software was built from source available at https://openindiana.org/.  The original community source was downloaded from  http://pub.mate-desktop.org/releases/1.22/mate-common-1.22.0.tar.xz
+This software was built from source available at https://openindiana.org/.  The original community source was downloaded from  http://pub.mate-desktop.org/releases/1.22/mate-common-1.22.2.tar.xz
 
 Further information about this software can be found on the open source community website at http://www.mate-desktop.org.
```